scripts/utils/clone-repos.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+# variables
+declare GROUP_NAME="<source-group-name>"
```

Comment:
Instead of hardcoding these. Can we pass them in via command line params and shArg?

Reply:
Added!